dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -132,8 +132,20 @@ public static ZenodoDoiResult registerZenodoDOI(ApiClient zenodoClient, Workflow
                 fillInMetadata(depositMetadata, workflow, workflowVersion);
             } catch (ApiException e) {
                 LOG.error("Could not create new deposition version on Zenodo. Error is " + e.getMessage(), e);
-                throw new CustomWebApplicationException("Could not create new deposition version on Zenodo."
+                if (e.getCode() == HttpStatus.SC_FORBIDDEN) {
```

Comment:
I suspect there could be other reasons for this status code. You could consider rewording the message to have "might" if so...

But not worth it; we can tackle that if it comes up.